modules/tensor_mechanics/test/tests/inclined_bc/inclined_bc_2d.i

```diff
@@ -35,28 +35,28 @@
     [../]
   [../]
   [./right_x]
-    type = PenaltyInclinedBC
+    type = PenaltyInclinedNoDisplacementBC
```

Comment:
I don't think it's really worth the trouble of having a non-Action version of these tests. Just have a 2D and a 3D test that use the Action.

Reply:
done